avocado/utils/memory.py

```diff
@@ -278,3 +279,35 @@ def get_buddy_info(chunk_sizes, nodes="all", zones="all"):
             buddyinfo_dict[chunk_size] += int(chunk_info)
 
     return buddyinfo_dict
+
+
+def set_thp_value(value, feature):
```

Comment:
would it not be better if the order be like `(feature, value)` as we generally denote key first, and value next ?